src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -338,41 +340,72 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
                 ${this._manifest?.version != null
                   ? html`<div class="version">${this._manifest.version}</div>`
                   : nothing}
+                ${this._manifest?.quality_scale &&
+                this._manifest?.quality_scale !== "internal"
+                  ? html`
+                      <div class="quality-scale integration-info">
```

Comment:
You should use `ha-list-item` so we are sure it aligns with the items underneath it.